requirements.txt

```diff
@@ -1 +1 @@
-python-consul==0.4.0
+python-consul==0.7.0
```

Comment:
maybe it makes sense to use less strict requirement like `>=0.7.0,<0.8.0` assuming that there are no breaking changes in 0.7.x line.

Travis now supports cron jobs and I could setup one so we will be notified on any build errors if any `python-consul` update breaks backwards compatibility.